README.md

```diff
@@ -485,7 +485,7 @@ With the receiver's approval of the transfer, the transaction can be signed and
 broadcast:
 ```sh
 bdk -n regtest wallet -w issuer -d "$DESC_TYPE($xprv_der_0)" \
-  sign --psbt $(base64 -w0 "data0/$PSBT")
+  sign --psbt $(cat data0/$PSBT | base64 | tr -d '\r\n')
```

Comment:
```suggestion
  sign --psbt $(base64 data0/$PSBT | tr -d '\r\n')
```

Reply:
mac not support base64 `file-path` ,will return invalid argument ,could we use `base64 -i` to replace it ?
```
echo "hello" > demo.txt
xxx@192 rgb-sandbox % base64 demo.txt 
base64: invalid argument demo.txt
Usage:	base64 [-hDd] [-b num] [-i in_file] [-o out_file]
  -h, --help     display this message
  -Dd, --decode   decodes input
  -b, --break    break encoded string into num character lines
  -i, --input    input file (default: "-" for stdin)
  -o, --output   output file (default: "-" for stdout)
xxx@192 rgb-sandbox % base64 -i demo.txt 
aGVsbG8K
```

Reply:
on Linux `-i` means `--ignore-garbage` (ignore non-alphabet characters) so we can't use that

let's keep the `cat` version then